bugsnag/event.py

```diff
@@ -254,6 +261,21 @@ def _generate_error_list(
                 )
             )
 
+        # unwrap BaseExceptionGroups so that their contained exceptions are
+        # also reported
+        # we don't recurse into nested BaseExceptionGroups or cause/context
+        # here because there's a big risk of that leading to a huge number of
+        # exceptions, which is difficult to reason about
+        if isinstance(exception, BaseExceptionGroup):
+            for sub_exception in exception.exceptions:  # type: ignore
```

Comment:
This `type: ignore` comment is because we run `mypy` on Python 3.5 (as well as 3.11) where it has no knowledge of `BaseExceptionGroup`, so complains about using `exception.exceptions`